### PR TITLE
C1189 salesforce connector feature

### LINKED ISSUE #7067: 'Salesforce' is misspelled in interface
Says SalesForce but it should say Salesforce :-)

Thank you for fixing! We're just starting to sell the Salesforce connector, so I hope this screen will get a lot of views soon.

vhamer___cartodb

### PR DESCRIPTION
**Pull request by javitonino**
Move Salesforce from feature flag to user/org feature. Edit texts for correct capitalization, closes #7067

Merge after central changes

Migration: #7887 (also here, will be removed from this branch via rebase)

### CODE REVIEW

**Comment by javitonino** [comment]
CR @gfiorav + @xavijam (bit of frontend work on this one)

**Comment by xavijam** [comment]
Frontend: 👍

File: db/migrate/20160616122125_add_salesforce_connector.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  up do
+    add_column :users,         :salesforce_datasource_enabled, :boolean, null: false, default: false
+    add_column :organizations, :salesforce_datasource_enabled, :boolean, null: false, default: false
+  end
+
+  down do
+    drop_column :users,         :salesforce_datasource_enabled
+    drop_column :organizations, :salesforce_datasource_enabled
+  end
+end
```
Comment (gfiorav) [inline db/migrate/20160616122125_add_salesforce_connector.rb]: Remember to separino

**Comment by gfiorav** [comment]
Backed is 📦 ready when migration is deployed and well 💃

**Comment by gfiorav** [comment]
Backed is 📦 ready when migration is deployed and well 💃

**Pull request merged**